guides/common/modules/con_unattended-use-customization-and-image-remastering.adoc

```diff
@@ -68,7 +68,10 @@ For example, `fdi.pxfactvalue1` sets the value for the fact named with `fdi.pxfa
 
 fdi.pxip, fdi.pxgw, fdi.pxdns::
 Manually configures IP address (`fdi.pxip`), the gateway (`fdi.pxgw`), and the DNS (`fdi.pxdns`) for the primary network interface.
-If your omit these parameters, the image uses DHCP to configure the network interface.
+If you omit these parameters, the image uses DHCP to configure the network interface.
+You can add multiple DNS entries in a comma-separated list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`
```

Comment:
```suggestion
You can add multiple DNS entries in a footnote:[comma-separated] list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`
```
However, I would move the footnote closer to the culprit.

Reply:
agreed, applied as 0e097b2143b256d1c81cb26d644144d9fe10ccff